filament/backend/include/private/backend/DriverAPI.inc

```diff
@@ -303,6 +303,7 @@ DECL_DRIVER_API_SYNCHRONOUS_0(bool, isProtectedContentSupported)
 DECL_DRIVER_API_SYNCHRONOUS_N(bool, isStereoSupported, backend::StereoscopicType, stereoscopicType)
 DECL_DRIVER_API_SYNCHRONOUS_0(bool, isParallelShaderCompileSupported)
 DECL_DRIVER_API_SYNCHRONOUS_0(bool, isDepthStencilResolveSupported)
+DECL_DRIVER_API_SYNCHRONOUS_N(bool, isDepthStencilBlitSupported, backend::TextureFormat, format)
```

Comment:
so depth blitting support is not all or nothing? it depends on the specific format?

Reply:
In practice, it's probably all or nothing. 
But in Vulkan, for the general use case of "can you do x with texture", it's  a check against the format of the texture.